config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -0,0 +1,34 @@
+
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const fs = require('fs');
+
+const isFile = filePath => (fs.existsSync(filePath) && !fs.lstatSync(filePath).isDirectory());
+
+const devSiteConfig = () => {
+  const processPath = process.cwd();
+  /* Get the root path of a mono-repo process call */
+  const rootPath = processPath.includes('packages') ? processPath.split('packages')[0] : processPath;
+
+  /* Get the site configuration to add as a resolve path */
+  let devSiteConfigPath = path.resolve(path.join(rootPath, 'dev-site-config'));
+  const customSiteConfigPath = path.join(devSiteConfigPath, 'site.config.js');
+  if (!isFile(customSiteConfigPath)) {
```

Comment:
Should we throw a console warning here to tell the developer their custom path is invalid?